Website/src/rendering/src/redux/ocCurrentCart/index.ts

```diff
@@ -233,8 +238,29 @@ export const retrieveCart = createOcAsyncThunk<RequiredDeep<DOrderWorksheet> | u
       ) {
         ThunkAPI.dispatch(retrievePayments(existingOrder.ID));
       }
-      ThunkAPI.dispatch(retrievePromotions(existingOrder.ID));
+      const existingOrderPromos = (await ThunkAPI.dispatch(retrievePromotions(existingOrder.ID)))
```

Comment:
The promos are not transferred with this scenario:
1. Log in
2. Complete an order until the Thank you page. (so you do not have any new empty order)
3. Log out
4. As an anonymous shopper, add products to cart, add promos, and go to the pre-checkout page.
5. From the pre-checkout page, log in.
Promos are not merged.

Reply:
Should be fixed now